### DISCUSSION TITLE
I need tap on item with text contains xpto.

### DISCUSSION THREAD

**Discussion by elNuketown** [mobile-dev-inc/Maestro#1903] [Q&A]
I have element with no has "id" on element, and your text on iOS, is value together on "balance" with my system.

**example: 
- tapOn: "Balance $ 1.011,11"** 

But i need tap only "Balance"

How to do it ?

**Comment by Fishbowler** [comment]
Unsure what you mean by "xpto" or "your text".

I've assumed that the balance changes, and that "Balance:" is the only static part.

To address this element, `tapOn: 'Balance.*'`
